maas/client/viscera/tests/test_machines.py

```diff
@@ -322,6 +323,16 @@ def test__release_with_wait_failed_disk_erasing(self):
             wait_interval=0.1
         )
 
+    def test__get_details(self):
+        machine = make_origin().Machine({
+            "system_id": make_name_without_spaces("system-id"),
+            "hostname": make_name_without_spaces("hostname"),
+        })
+        data = machine.get_details()
```

Comment:
Does this test actually work? As your not mocking the handler for it to return data for the `details` call.